app/models/user.rb

```diff
@@ -6,8 +6,8 @@ class User < ApplicationRecord
 
   validates :name, presence: true
 
-  def total_fines
-    total = checkouts.inject(0) { |total, checkout| total + checkout.fine }
-    total = sprintf('%.2f', total)
-  end
+  # def total_fines
```

Comment:
Did you mean to leave this commented out? Typically we'd delete rather than comment out code. If we need it at a later date, we can awlays get it back from git history.